src/stratigraphy/depthcolumn/depthcolumn.py

```diff
@@ -130,10 +174,21 @@ def is_valid(self) -> bool:
     def identify_groups(
         self,
         description_lines: list[TextLine],
-        geometric_lines: list[Line],
-        material_description_rect: fitz.Rect,
+        geometric_lines: list[Line],  # TODO: Parameter not used. Shall we keep?
+        material_description_rect: fitz.Rect,  # TODO: Parameter not used. Shall we keep?
```

Comment:
These are used in the other implementation of this method, the one from BoundaryDepthColumn, not LayerDepthColumn. And since they both need to conform to the signature of the same abstract method from the parent class DepthColumn, these parameter are also required here. Of do you know of a better way to deal with this situation?